File: app/src/main/java/com/example/android/moviesremake/realm/InitRealm.java
package com.example.android.moviesremake.realm;

import android.app.Application;

import io.realm.Realm;
import io.realm.RealmConfiguration;

/**
 * Created by PepovPC on 1/14/2018.
 */

public class InitRealm extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        Realm.init(this);
        RealmConfiguration realmConfig = new RealmConfiguration.Builder()
                .name("tasky.realm")
                .schemaVersion(0)
                .build();
        Realm.setDefaultConfiguration(realmConfig);
    }

}

File: app/src/main/java/com/example/android/moviesremake/realm/InitRealm.kt
package com.example.android.moviesremake.realm

import android.app.Application

import io.realm.Realm
import io.realm.RealmConfiguration

/**
 * Created by PepovPC on 1/14/2018.
 */

class InitRealm : Application() {

    override fun onCreate() {
        super.onCreate()
        Realm.init(this)
        val realmConfig = RealmConfiguration.Builder()
                .name("tasky.realm")
                .schemaVersion(0)
                .build()
        Realm.setDefaultConfiguration(realmConfig)
    }

}